docs/tutorials/edx-platform.rst

```diff
@@ -0,0 +1,124 @@
+
+    $ tutor mounts list
+    - name: /home/data/regis/projets/overhang/repos/edx/edx-platform
+    build_mounts:
+    - image: openedx
+        context: edx-platform
+    - image: openedx-dev
+        context: edx-platform
+    compose_mounts:
+    - service: lms
+        container_path: /openedx/edx-platform
+    - service: cms
+        container_path: /openedx/edx-platform
+    - service: lms-worker
+        container_path: /openedx/edx-platform
+    - service: cms-worker
+        container_path: /openedx/edx-platform
+    - service: lms-job
+        container_path: /openedx/edx-platform
+    - service: cms-job
+        container_path: /openedx/edx-platform
+
+Working on edx-platform Python dependencies
+-------------------------------------------
+
+Quite often, developers don't want to work on edx-platform directly, but on a dependency of edx-platform. For instance: an XBlock. This works the same way as above. Let's take the example of the `"edx-ora2" <https://github.com/openedx/edx-ora2>`__ package, for open response assessments. First, clone the Python package::
+
+    cd /my/workspace/edx-ora2
+    git clone https://github.com/openedx/edx-ora2 .
+
+Then, check out the right version of the package. This is the version that is indicated in the ``edx-platform/requirements/edx/base.txt``. Be careful that the version that is currently in use in your version of edx-platform is **not necessarily the latest version**.
+
+    git checkout <my-version-tag-or-branch>
+
+Then, mount this repository::
+
+    tutor mounts add /my/workspace/edx-ora2
+
+Verify that your repository is properly bind-mounted by running ``tutor mounts list``::
+
+    $ tutor mounts list
+    - name: /my/workspace/edx-ora2
+    build_mounts:
+    - image: openedx
+        context: req-edx-ora2
+    - image: openedx-dev
+        context: req-edx-ora2
+    compose_mounts:
+    - service: lms
+        container_path: /openedx/requirements/edx-ora2
+    - service: cms
+        container_path: /openedx/requirements/edx-ora2
+    - service: lms-worker
+        container_path: /openedx/requirements/edx-ora2
+    - service: cms-worker
+        container_path: /openedx/requirements/edx-ora2
+    - service: lms-job
+        container_path: /openedx/requirements/edx-ora2
+    - service: cms-job
+        container_path: /openedx/requirements/edx-ora2
+
+It is quite possible that your package is not automatically recognized and bind-mounted by Tutor. In such a case, you will need to create a :ref:`Tutor plugin <plugin_development_tutorial>` that implements the :py:data:`tutor.hooks.Filters.EDX_PLATFORM_PYTHON_PACKAGES` patch::
+
+    tutor.hooks.Filters.EDX_PLATFORM_PYTHON_PACKAGES.add_item("my-package")
+
+After you implement and enable that plugin, ``tutor mounts list`` should display your directory among the bind-mounted directories.
+
+You should then re-build the "openedx" Docker image to pick up your changes::
+
+    tutor images build openedx
```

Comment:
```suggestion
    tutor images build openedx-dev
```

In order to apply to ``tutor dev start``, I believe you'll need to specifically build the dev image.